quanta-d51-2u/templates/flash_bios.sh

```diff
@@ -46,3 +46,6 @@ curl -T ./${BACKUP_FILE} <%= api.files %>/<%= nodeId %>-${VERSION}
 # Flash new image
 ${CMD} <%= sku.biosFirmware.args %>
 
+# Wait some time for the internal process to finish
+# otherwise catalog ami task after flashing will hang the node in some fw version (like 3A19)
+sleep 30
```

Comment:
Rather than a sleep, can we parse the output of the catalog command here to wait for the process to complete?  This would align with what you did for the BMC here: https://github.com/RackHD/on-skupack/blob/master/quanta-d51-2u/templates/flash_bmc.sh#L63